modules/nf-core/gatk4/splitncigarreads/main.nf

```diff
@@ -46,4 +46,15 @@ process GATK4_SPLITNCIGARREADS {
         gatk4: \$(echo \$(gatk --version 2>&1) | sed 's/^.*(GATK) v//; s/ .*\$//')
     END_VERSIONS
     """
+
+    stub:
```

Comment:
This one should actually be fixed at the nf-core repo or?

Reply:
Yes, it should, as it is an nf-core module

Reply:
Looks like this is fixed over at the nf-core repo. What is the correct way to fix it here as well. `nf-core modules update` within tomte, selecting that module and then pushing the changes?

Reply:
I'll try that way. If not right I'll revert later

Reply:
Updated

Reply:
Now that it has been updated in nf-core, you should update the module in the pipeline, which should change the modules.json. If a change is made outside of the nf-core/modules, a patch would have to be made, to keep the modules in line to the nf-core ones

Reply:
This module had already been fixed in nf-core, I just needed to update it here in Tomte as well (running `nf-core modules update`). So no external work needed in this case I think.

The `modules.json` includes the diff you mention.

I forgot the CHANGELOG though, thank you for pointing that out 🙏  Updated now.

Can you have a look to check that things are in order now?